src/collections/_documentation/common/getting_started/getting_started_template.md

```diff
@@ -0,0 +1,124 @@
+
+In the right place? We also offer documentation for:
+<!-- 
+Guideline: Add icons for related languages/frameworks.
+-->
+What's covered on this page:
+
+- [Install](#install)
+- [Initialize](#initialize)
+- [Verify Setup](#verify-setup)
+- [Capture Errors](#capture-errors)
+    1. [Enrich Error Data](#enrich-error-data) 
+    2. [Set the Release Version](#set-release-version)
+    
+<!--
+Guideline: Add any step to set up information that's specific to the SDK; for example, in the JavaScript example, this heading is for Source Maps. Then add the link to that heading
+-->
+
+- [Monitor Performance](#monitor-performance)
+
+## Install {#install}
+
+1.  If you already have an account, log in to Sentry. If not, establish your account:
+    1. Use your Google, GitHub, or Azure Devops account
+    2. Or, provide the basics: Your name, email, and organization, then set your password.
+2. Select <SDK> as the platform you're working with
+3. Name your project
+
+*Projects* in Sentry scope events to a distinct application in your organization, and assign responsibility and ownership. 
+
+4. Choose your installation method:
+
+<!--
+Guideline: Add SDK specific installation information
+-->
+
+## Initialize {#initialize}
+
+`init` Sentry as soon as possible:
+<!--
+Guideline: Add init code example for this SDK 
+-->
+Once this is done, all unhandled exceptions are automatically captured by Sentry. 
+
+**Important: Record your DSN.** The *DSN* (Data Source Name) tells the SDK where to send the error, associating errors with the project you just created. If you forget your DSN, log in to Sentry to obtain the project DSN from your organization's *Settings -> Projects -> Client Keys (DSN)* in the Sentry web UI. The DSN includes the protocol, public key, server address, and project identifier, using this format:
+
+`https://<Public Key>@<Sentry Server Address>/<Project Identifier>`
+
+## Verify Setup {#verify-setup}
+
+Verify your setup by intentionally sending an error that breaks your application. Sentry will send an alert to the email address configured on [Sentry.io](http://sentry.io/) notifying you about the event.
+
+An *event* is one instance of sending data to Sentry. Generally, this data is an error. An *issue* is a grouping of similar events.
+
+For example, in <!--SDK name-->, 
+<!--
+Guideline: Add verify setup example for the SDK you are documenting
+-->
+Resolve the created error by either: 
+
+- Clicking `View on Sentry` in the alert sent to the email provided when you established your account
+- Logging in to Sentry, then opening your project
+
+On the **Issue Details** page, scroll down to the Exception stack trace and view the error. 
+
+## Capture Errors {#capture-errors}
+
+The reporting of an *event* - an error, exception, or rejection - is called *capturing*. When an event is captured, it’s sent to Sentry, then creates a new issue group (or is added to an already existing one) based on Sentry’s grouping algorithm. 
```

Comment:
same thing here, rejection means nothing

Reply:
also its issue, not issue group

Reply:
also it might not create a new issue, issues are just representation of unique errors. imo when we try to give too manhy details too much we will end up giving details incorrectly.

Reply:
I pulled that from some old content, so it's good to know we should refine away from it. I'll simplify.